neural_network_ploting_with_bp_algorithm_and_without_algorithm

```diff
@@ -0,0 +1,457 @@
+#Neural Network Ploting with BP Algorithm and without Algorithm
```

Comment:
Hello, the first line should start with a single hashtag #, followed by a single whitespace, and then the title.

Reply:
Thank you. I have solved this problem.